src/tools/cephfs/DataScan.cc

```diff
@@ -446,7 +452,28 @@ int DataScan::scan_inodes()
   float progress = 0.0;
   librados::NObjectIterator i = data_io.nobjects_begin(n, m);
 #else
-  librados::NObjectIterator i = data_io.nobjects_begin();
+  librados::NObjectIterator i;
+  bool legacy_filtering = false;
```

Comment:
these #ifdefs aren't functional, just aide memoires for me when the post-infernalis sharded pgls stuff goes in.